CHANGELOG.md

```diff
@@ -15,6 +15,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Fix nil pointer exception when running an incremental backup on SharePoint where the base backup used an older index data format.
 - --user and --mailbox flags (already not supported) have been removed from CLI examples for details and restore commands.
 - Improve restore time on large restores by optimizing how items are loaded from the remote repository.
+- Fix handling of duplicate folders in Exchange. Duplicate folders will be merged during restore operations.
+
+### Known Issues
+- Restore operations will merge duplicate Exchange folders into a single folder.
```

Comment:
Might want to further explain `duplicate folders` here.  It's not easy to know what that means in this sentence (duplicate as in duplicate contents, duplicate names, exact duplication due to server error or just display name duplication, etc).

Also: issue, or feature addition?  outlook "doesn't allow" duplicate folders anyway.  So in a sideways way, this kinda feels like an addition, to me.

Reply:
>issue, or feature addition?

That's a good question. I guess I'm looking at it from the perspective of data out doesn't match data in which could be a little surprising to users so a "known issue" seems to cover that?